diagnostic.md

```diff
@@ -36,19 +36,22 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
+The path listed above is a relative path because it tells you exactly where to go from the Home directory. It gives a direct route, avoiding online URLs that may change over time and therefore lose the information they linked to.
+
+
 
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+We would use an absoulte path because that's where the font is being hosted by the server. It similar to the argument made in class: any photo hosted by google isnt going anywhere, we'll be able to link to it as long is its being hosted.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Depends on the situation, both have different stengths and weaknesses. For example: If we are working with others, it should have an absoulte path so that eveyone can access the files individually and not have to rely on one person to host them in a certain location. 
  <!-- Answer Ends Here -->
```

Comment:
In this scenario, _we_ are using an image file inside our project, so we would use a relative path to refer to it within our directory. If someone else was trying to use our file, the file would have to be hosted on the internet somewhere so that it would be accessible to them.